main.html

```diff
@@ -1,51 +1,67 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/assets/css/main.css" />
+  </head>
 
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-  <link rel="stylesheet" href="/assets/css/main.css">
-</head>
-
-<body>
-  <div class="container">
-    <!-- header Nhi-->
-    <div class="header">
-                <a href="" class="business"><img src="/assets/img/logo.png" width="108"></a>
-                <a href="" class="menu">ABOUT</a>
-                <a href="" class="menu">WORK</a>
-                <a href="" class="menu">TEAM</a>
-                <a href="" class="menu">BLOG</a>
-                <a href="" class="menu">CONTACT</a>
+  <body>
+    <div class="container">
+      <!-- header Nhi-->
+      <div class="header">
+        <a href="" class="business"
+          ><img src="/assets/img/logo.png" width="108"
+        /></a>
+        <a href="" class="menu">ABOUT</a>
+        <a href="" class="menu">WORK</a>
+        <a href="" class="menu">TEAM</a>
+        <a href="" class="menu">BLOG</a>
+        <a href="" class="menu">CONTACT</a>
         <span>
-
-                <a href="" class="contact">CONTACT US</a>
+          <a href="" class="contact">CONTACT US</a>
         </span>
-        </div>
+      </div>
 
-    <!--banner => Ly-->
-    <div class="banner">
-      <div class="banner-content">
-        <p class="text-heading">Grow your business.</p>
-        <p class="text-paragraph">Give your business a boost with a beautifully crafted homepage.
-          <br>
-        </p>
-        <a href="#" class="banner-button">
-          <div class="button">Learn more</div>
+      <!--banner => Ly-->
+      <div class="banner">
+        <div class="banner-content">
+          <p class="text-heading">Grow your business.</p>
+          <p class="text-paragraph">
+            Give your business a boost with a beautifully crafted homepage.
+            <br />
+          </p>
+          <a href="#" class="banner-button">
+            <div class="button">Learn more</div>
+          </a>
+        </div>
+      </div>
+      <!-- main -->
+      <div class="main-content"></div>
+      <!-- about us -->
+      <div class="about-us"></div>
+      <!--footer Nhi-->
+      <div class="footer">
+        <div>
+          <p class="Grow">Grow your business.</p>
+          <p class="Today">
+            Today is the day to build the business of your dreams. Share your
+            mission with <br />
+            the world — and blow your customers away.
```

Comment:
please keep this code in one line

Reply:
I think break this code is more easier to read

Reply:
No, you should to keep the code in one line for performance